rake/spec/default/60-st2_all-services-ok_spec.rb

```diff
@@ -9,17 +9,29 @@
     subject { host(spec[:rabbitmqhost]) }
     it { is_expected.to be_reachable }
   end
+  describe 'rabbitmq' do
+    subject { host(spec[:rabbitmqhost]) }
+    it { should be_reachable.with( :port => 5672, :timeout => 1 ) }
```

Comment:
not sure about the `expected` syntax, that I've provided the correct example, but anyway it's preferred to use.